config/rbac.yml

```diff
@@ -24,17 +24,33 @@ rules:
 - apiGroups: ["install.package.carvel.dev"]
   resources: ["installedpackages", "installedpackages/status"]
   verbs: ["*"]
-- apiGroups: ["package.carvel.dev"]
-  resources: ["packages", "packages/status"]
-  verbs: ["*"]
 - apiGroups: ["install.package.carvel.dev"]
   resources: ["packagerepositories", "packagerepositories/status"]
   verbs: ["*"]
+- apiGroups: ["install.package.carvel.dev"]
+  resources: ["internalpackages"]
+  verbs: ["*"]
+- apiGroups: ["package.carvel.dev"]
+  resources: ["packages", "packages/status"]
+  verbs: ["*"]
 #! TODO Needed for kapp-controller when deploying
 #! repository App CR (temporarily needed only for POC)
 - apiGroups: [""]
   resources: ["configmaps"]
   verbs: ["*"]
+#! Aggregated API server permissions
+- apiGroups: ["apiregistration.k8s.io"]
+  resources: ["apiservices"]
+  verbs: ["update", "get"]
+- apiGroups: [""]
+  resources: ["namespaces"]
+  verbs: ["list", "watch"]
+- apiGroups: ["admissionregistration.k8s.io"]
+  resources: ["mutatingwebhookconfigurations"]
+  verbs: ["list", "watch"]
+- apiGroups: ["admissionregistration.k8s.io"]
+  resources: ["validatingwebhookconfigurations"]
+  verbs: ["list", "watch"]
```

Comment:
do we need webhook perms?

Reply:
Yea, the webserver won't start if it can't watch these resources